js_modules/dagit/packages/core/src/asset-graph/useLiveDataForAssetKeys.tsx

```diff
@@ -20,6 +20,7 @@ const SUBSCRIPTION_MAX_POLL_RATE = 2 * 1000;
  */
 export function useLiveDataForAssetKeys(assetKeys: AssetKeyInput[]) {
   const liveResult = useQuery(ASSETS_GRAPH_LIVE_QUERY, {
+    fetchPolicy: 'cache-first', // todo bengotow: Is this correct?
```

Comment:
This one could be cache-and-network as well

It's odd, actually... I think it'd be pretty noticeable if this was doing cache-first, but I guess we explicitly call .refresh() using that refreshAtInterval hook, so maybe that was patching it enough you didn't notice it loaded stale.